src/components/modules/saver.ts

```diff
@@ -28,15 +28,11 @@ export default class Saver extends Module {
    * @returns {OutputData}
    */
   public async save(): Promise<OutputData> {
-    const { BlockManager, Tools, ModificationsObserver } = this.Editor;
+    const { BlockManager, Tools } = this.Editor;
     const blocks = BlockManager.blocks,
         chainData = [];
 
     try {
-      /**
-       * Disable onChange callback on save to not to spam those events
```

Comment:
Do you mean that this problem is not relevant for now? Why?

Reply:
Its not related to the delimiter because when we call `save` function inside OnChange event it disables upcomming onchange events by calling `modificationObserver.disable()`.

Reply:
@neSpecc do you have any test cases such that I can check because I don't know if it occurs or not

> Do you mean that this problem is not relevant for now? Why?